internal/daemon/api_services.go

```diff
@@ -180,12 +180,26 @@ func v1PostServices(c *Command, r *http.Request, _ *userState) Response {
 	// Use the original requested service name for the summary, not the
 	// resolved one. But do use the resolved set for the count.
 	var summary string
-	if len(services) == 1 {
+	switch len(services) {
+	case 0:
```

Comment:
Should we not instead check if the task set is empty? Has the same outcome, but we shouldn't rely on the service names used for summary generation as someone might add an operation that can affect more than services.

Reply:
Fair - done